x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/apikey/CrossClusterApiKeyRoleDescriptorBuilder.java

```diff
@@ -121,9 +124,9 @@ static void validate(RoleDescriptor roleDescriptor) {
             throw new IllegalArgumentException("remote cluster permissions must be empty");
         }
         final String[] clusterPrivileges = roleDescriptor.getClusterPrivileges();
-        if (false == Arrays.equals(clusterPrivileges, CCS_CLUSTER_PRIVILEGE_NAMES)
-            && false == Arrays.equals(clusterPrivileges, CCR_CLUSTER_PRIVILEGE_NAMES)
-            && false == Arrays.equals(clusterPrivileges, CCS_AND_CCR_CLUSTER_PRIVILEGE_NAMES)) {
+        // must contain either "cross_cluster_search" or "cross_cluster_replication" or both
+        if ((Arrays.asList(clusterPrivileges).contains("cross_cluster_search")
+            || Arrays.asList(clusterPrivileges).contains("cross_cluster_replication")) == false) {
```

Comment:
Discovered this as latent bug when updating the test to be accurate to pre 8.14 API keys. Since we (me) added a new cluster privilege in 8.14, the elder check would fail anytime we updated the set of privilges since it compares the set of privilges in the version where the API was created vs. the set of privileges for the current cluster.